src/components/StudyInfo/CavaticaCounts.js

```diff
@@ -19,7 +25,21 @@ const CavaticaCounts = ({projects, title, hideIcon}) => {
       <List.Item
         as={Link}
         to={`/study/${title}/cavatica`}
-        onClick={e => e.stopPropagation()}
+        onClick={e => {
+          e.stopPropagation();
+          logEvent(
+            inheritedEventProps.scope
+              ? inheritedEventProps.scope + '__TOOLTIP_PROJECTS__CLICK'
```

Comment:
Why not use an array to describe the scope like [this](https://github.com/amplitude/react-amplitude#example-instrumenting-tic-tac-toe-from-facebooks-intro-to-react-tutorial)?

Reply:
thought about that. I think we get the same level of information by having a `view` prop in every event that is the name of the current view from the route then `scope` just applies to the specific container in the view. I think this also makes it easier to query from the Amplitude dashboard(s) as you don't have to check for something in an array. 

If we do use scope as an array it would be a better candidate to roll into the AmplitudeProxy to pass with every call